templates/page.html

```diff
@@ -273,12 +275,12 @@ <h3>Developers</h3>
         <div class="release-news">
           <img src="{{ '/static/img/release.png'|asseturl }}" />
           <div class="release-news-body">
-            <h3>REANA 0.7.2</h3>
-            <h6>4 February 2021</h6>
+            <h3>REANA 0.7.3</h3>
+            <h6>22 March 2021</h6>
             <p>
-              Minor update for REANA administrators. Adds deployment options to disable user sign-up form and user email confirmation.
+              Adds REANA-Client validation of input parameters and environment images and improves overall cluster resilience and support of compute backends.
```

Comment:
We could split into several phrases

Adds reana-client validation of workflow input parameters and environment images. Improves cluster resilience in case of job failures. Fixes HTCondor and Slurm integration for complex job commands.

(Hope it's not that long... If yes, we could end up with "Improves HTCondor and Slurm job passing" or some such)